profiles/permissions.easy

```diff
@@ -316,3 +316,10 @@
 
 # postfix (bsc#1201385)
 /usr/sbin/postlog                                       root:maildrop 2755
+
+# kismet (bsc#1200954 bsc#1207654)
```

Comment:
It's a bit picky but could you please place a comma between the bugs in parantheses? Then we maintain a common style in the file.

Reply:
Thanks for the quick review. Done!